### PR TITLE
Biogenerator and botany bottle improvements

### PR DESCRIPTION
**Pull request by erwgd**
Black pepper is a plant product but no plant in the game exists to produce it, so for now the biogenerator can create it. It may be produced to either fill the container attached to the biogenerator or to its own container (a 20u pepper mill) at no extra cost.

Empty bottles can be produced. They have the same volume and reagent transfers as the fertilizer bottles. You can now avoid mixing up bottles of mutagen with actual fertilizer bottles.

Additionally, the weed killer and pest killer are now under the new 'killer' branch of bottles, with the same defines as the fertilizer bottles. This allows them to spawn with 50 units as intended. The biomass cost for these items has not changed.

I removed capitalizations in some of the biogenerator options. I added descriptions to each botany bottle, except for the empty bottle and bottle/killer.

:cl: Erwgd
bugfix: Weed Killer bottles and Pest Killer bottles now contain 50 units.
rscadd: The biogenerator can now make empty bottles and black pepper.
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
Neat.

File: code/modules/hydroponics/biogenerator.dm [hunk 1]
```diff
@@ -258,10 +268,13 @@
...
 		if("wk")
 			if (check_cost(50/efficiency)) return 0
-			else new/obj/item/weapon/reagent_containers/glass/bottle/weedkiller(src.loc)
+			else new/obj/item/weapon/reagent_containers/glass/bottle/killer/weedkiller(src.loc)
 		if("pk")
 			if (check_cost(50/efficiency)) return 0
-			else new/obj/item/weapon/reagent_containers/glass/bottle/pestkiller(src.loc)
+			else new/obj/item/weapon/reagent_containers/glass/bottle/killer/pestkiller(src.loc)
+		if("empty")
+			if (check_cost(5/efficiency)) return 0
```
Comment (KorPhaeron) [inline code/modules/hydroponics/biogenerator.dm]: No same line returns

File: code/modules/hydroponics/biogenerator.dm [hunk 2]
```diff
@@ -234,6 +237,10 @@
 			if(check_container_volume(10)) return 0
 			else if (check_cost(20/efficiency)) return 0
 			else beaker.reagents.add_reagent("milk",10)
+		if("bpepper")
+			if(check_container_volume(10)) return 0
```
Comment (KorPhaeron) [inline code/modules/hydroponics/biogenerator.dm]: No same line returns

File: code/modules/hydroponics/biogenerator.dm [hunk 3]
```diff
@@ -141,8 +141,10 @@
 			dat += "<div class='statusDisplay'>"
 			dat += "10 milk: <A href='?src=\ref[src];create=milk;amount=1'>Make</A><A href='?src=\ref[src];create=milk;amount=5'>x5</A> ([20/efficiency])<BR>"
 			dat += "10 cream: <A href='?src=\ref[src];create=cream;amount=1'>Make</A><A href='?src=\ref[src];create=cream;amount=5'>x5</A> ([30/efficiency])<BR>"
-			dat += "Milk Carton: <A href='?src=\ref[src];create=cmilk;amount=1'>Make</A><A href='?src=\ref[src];create=cmilk;amount=5'>x5</A> ([100/efficiency])<BR>"
-			dat += "Cream Carton: <A href='?src=\ref[src];create=ccream;amount=1'>Make</A><A href='?src=\ref[src];create=ccream;amount=5'>x5</A> ([300/efficiency])<BR>"
+			dat += "Milk carton: <A href='?src=\ref[src];create=cmilk;amount=1'>Make</A><A href='?src=\ref[src];create=cmilk;amount=5'>x5</A> ([100/efficiency])<BR>"
+			dat += "Cream carton: <A href='?src=\ref[src];create=ccream;amount=1'>Make</A><A href='?src=\ref[src];create=ccream;amount=5'>x5</A> ([300/efficiency])<BR>"
+			dat += "10 black pepper: <A href='?src=\ref[src];create=bpepper;amount=1'>Make</A><A href='?src=\ref[src];create=bpepper;amount=5'>x5</A> ([25/efficiency])<BR>"
```
Comment (KorPhaeron) [inline code/modules/hydroponics/biogenerator.dm]: Shouldn't it say 10u of black pepper or something?

**Pull request merged**